docs/docs/modeling.md

```diff
@@ -118,4 +118,28 @@ classDiagram
 
 ## AssignedCategory -　
 
-## AssignedTask -
+## AssignedTask - 割当タスク
+
```

Comment:
I'd appreciate it if you could add the description/explanation about this `AssignedTask`.